pages/nettest.js

```diff
@@ -151,18 +151,10 @@ export default class extends React.Component {
         </Head>
         <OONIRunHero href={'https://run.ooni.io'} />
         <Container p={4}>
-          <Text pt={2}>Measure internet censorship by running OONI Probe.</Text>
 
-          <Heading pt={2} h={2}>You have the app</Heading>
-          <Text pt={2} pb={3}>If you have the OONI Probe app installed, click Run to start the test.</Text>
-
-          <Link href={deepLink}>
-            <Button>Run</Button>
```

Comment:
The deep link must remain, because in the event that it does not trigger mobile deep linking because we failed to detect that they are on mobile (or the plethora of ways in which it might fail), you need to be able to override it with a clickable button which has the `ooni://` prefix.

Reply:
Oh, thanks for the heads-up! I've kept this text, and made some edits.